app/src/main/java/com/example/upeers/ui/message/MessageUserInfoAdapter.kt

```diff
@@ -25,6 +30,9 @@ open class MessageUserInfoAdapter(private val myDataSet: List<MessageUserInfo>)
         val cardView = LayoutInflater.from(parent.context)
             .inflate(R.layout.layout_message_contact_card, parent, false) as View
 
+//        cardView.setOnClickListener {
+//        }
```

Comment:
comment should start from the current indent level

Reply:
do you have any idea how to fix this on android studio?